src/services/transaction.service.ts

```diff
@@ -30,10 +31,62 @@ export const postTransaction = async (session: Session, companyNumber: string, d
   const castedSdkResponse: Resource<Transaction> = sdkResponse as Resource<Transaction>;
 
   if (!castedSdkResponse.resource) {
-    throw createAndLogError(`Transaction API returned no resource for company number ${companyNumber}`);
+    throw createAndLogError(`Transaction API POST request returned no resource for company number ${companyNumber}`);
   }
 
   logger.debug(`Received transaction ${JSON.stringify(sdkResponse)}`);
 
   return castedSdkResponse.resource;
 };
+
+/**
+ * Response can contain a URL to start payment session if payment is needed
+ */
+export const closeTransaction = async (session: Session, companyNumber: string, transactionId: string): Promise<string | undefined> => {
+  const apiResponse: ApiResponse<Transaction> = await putTransaction(session, companyNumber, transactionId, DESCRIPTION, REFERENCE, transactionStatuses.CLOSED);
```

Comment:
I've gone with the default transaction description and reference here:
DESCRIPTION = "Confirmation Statement Transaction";
REFERENCE = "ConfirmationStatementReference";
not sure if this should be something more specific?